rinja_parser/src/tests.rs

```diff
@@ -371,8 +371,31 @@ fn change_delimiters_parse_filter() {
         expr_end: "=}",
         ..Syntax::default()
     };
-
     Ast::from_str("{= strvar|e =}", None, &syntax).unwrap();
+
+    let syntax = Syntax {
+        expr_start: "🖎", // U+1F58E == b"\xf0\x9f\x96\x8e"
```

Comment:
Nice :+1:

Please just add a comment on this one explaining it ensures that unicode characters can be used as expression markers.

Reply:
Do you think that a speaking name is enough?

Reply:
Yep, the function name makes it obvious for me at least.